sdk/servicebus/azure-messaging-servicebus/src/samples/README.md

```diff
@@ -9,48 +9,62 @@ urlFragment: servicebus-samples
 ---
 
 # Azure Service Bus Samples client library for Java
-This document explains samples and how to use them.
+Azure Service Bus samples are a set of self-contained Java programs that demonstrate interacting with Azure Service Bus
+using the client library. Each sample focuses on a specific scenario and can be executed independently.
 
 ## Key concepts
-Key concepts are explained in detail [here][SDK_README_KEY_CONCEPTS].
+Key concepts are explained in detail [here][sdk_readme_key_concepts].
 
 ## Getting started
-Getting started explained in detail [here][SDK_README_GETTING_STARTED].
- 
-### Adding the package to your project
-
-Maven dependency for Azure app configuration Client library. Add it to your project's pom file.
-
-[//]: # ({x-version-update-start;com.azure:azure-messaging-servicebus;current})
-```xml
-<dependency>
-    <groupId>com.azure</groupId>
-    <artifactId>azure-messaging-servicebus</artifactId>
-    <version>7.0.0-beta.1</version>
-</dependency>
-```
-[//]: # ({x-version-update-end})
+Please refer to the [Getting Started][sdk_readme_getting_started] section.
+
+### Obtaining an Event Hub instance connection string
+
+Most of the samples authorize with Service Bus using a connection string generated for that Service Bus namespace. The
+connection string value can be obtained by:
+
+1. Going to your Service Bus namespace in Azure Portal.
+1. Go to "Shared access policies" for your Service Bus namespace.
+1. Click on the "RootManageSharedAccessKey" policy.
+1. Copying the connection string from the policy's properties.
```

Comment:
Why they are all `1` ?

Reply:
Because markdown will automatically number them.

Reply:
It's part of the protocol so you don't have to renumber them manually.
https://github.com/conniey/azure-sdk-for-java/tree/add-readme/sdk/servicebus/azure-messaging-servicebus/src/samples